services/bettersocial/fcmToken/sendMultiDeviceNotification.js

```diff
@@ -24,10 +24,11 @@ const sendMultiDeviceNotification = async (
           user_id: userIdFollowed,
           user_id_follower: userIdFollower,
           username_follower: userNameFollower
-        }
+        },
+        token: user?.token
       };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `sendToDevice` to `send` is a significant one. The `send` method requires a different payload structure compared to `sendToDevice`. In the new code, it seems like you're still using the old payload structure which might not work with the `send` method. Please ensure that the payload is correctly structured for the `send` method.

```diff
-        },
-        token: user?.token
-       };
+        }
+       };
+       payload.token = user?.token;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->